test/test_acoustid_tagger.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from headphones2.postprocess.taggers.acoustid_tagger import AcoustIDAlbumTagger
+from headphones2.postprocess.process_folder import FolderIterator
+from beets.library import Item
+from conftest import vcr
+
+HUMAN_EQUATION_MBID = '90392b89-77fc-3c39-813d-c5c09abf8943'
+DARK_MEDICINE_ACOUSTID = 'f7c6418c-d8a7-46f8-bbeb-d5a11481776e'
+
+@pytest.fixture
+def single_item():
+    track_22 = r"C:\Users\Omer\Desktop\The Theory of Everything [2013]\20 Ayreon - The Theory of Everything [2013] - Phase 2_ Symmetry, Dark Medicine.flac"
```

Comment:
Maybe some open-source music file which is properly detected by both them, and add it to the git repo so everyone can run the tests.